build-tools/packages/build-cli/docs/exec.md

```diff
@@ -11,32 +11,42 @@ Run a shell command in the context of a package or release group.
 
 ```
 USAGE
-  $ flub exec CMD [-v] [-a | -d <value> | --packages | -g
-    client|server|azure|build-tools|gitrest|historian] [--releaseGroupRoots] [--private] [--scope <value> | -g
-    client|server|azure|build-tools|gitrest|historian]
+  $ flub exec CMD [-v | --quiet] [--concurrency <value>] [--releaseGroupRoot
+    client|server|azure|build-tools|gitrest|historian|all | [--all | --dir <value> | --packages | -g
+    client|server|azure|build-tools|gitrest|historian|all] |  | ] [--private] [--scope <value> | --skipScope <value>]
```

Comment:
Unrelated nit: something funky seems to be happening when this is generated. `|all] | | ]` ? I see it in other md files too for in the same place.